microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -306,13 +429,18 @@ WebView2 can specify a default background color. This can be any opaque color or
 <!-- ====================================================================== -->
 ## Visual hosting
 
-In visual hosting, content is embedded to a given location on the application. This location must handle how content will scale and behave in an app when the user interacts with the application.  In addition to the window management described for windowed hosting, visual hosting will need the app to manage the composition-based rendering, when it receives any user interactions.  For more information about visual hosting, see [Using the Visual layer in desktop apps](/windows/apps/desktop/modernize/visual-layer-in-desktop-apps).
+In Visual hosting, content is embedded to a given location on the application. This location must handle how content will scale and behave in an app when the user interacts with the application.  In addition to the window management described for Windowed hosting, Visual hosting will need the app to manage the composition-based rendering, when it receives any user interactions.  For more information about Visual hosting, see [Using the Visual layer in desktop apps](/windows/apps/desktop/modernize/visual-layer-in-desktop-apps).
 
-If your WebView2 app uses visual hosting:
+If your WebView2 app uses Visual hosting:
 
 * Inputs are routed to the app's `HWND` and must be configured to send the spatial input (for example, mouse, touch, and pen) based on positions, _not_ what currently has focus, such as a keyboard.
 
-The following APIs can be used when configuring WebView2 in a visual hosting environment:
+
+<!-- ------------------------------ -->
+#### APIs for Visual hosting
+
+The following APIs can be used when configuring WebView2 in a Visual hosting environment:
+<!-- todo: are the below tab lists a straight copy of "Overview of APIs"?  link to there instead? -->
```

Comment:
are the below tab lists a straight copy of "Overview of APIs"?  link to there instead?

https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/windowed-vs-visual-hosting?branch=pr-en-us-3271#apis-for-visual-hosting
Overview of APIs:
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3271

Reply:
added links in hidden comments but have to move them outside of comment to click them

Reply:
Outdated, entered fresh Comment